src/spawn_players.c

```diff
@@ -22,6 +22,50 @@
 #include "menus.h"
 #include "render_player.h"
 
+f32 D_80165210[8];
+f32 D_80165230[7];
+f32 D_8016524C;
```

Comment:
It's possible that D_8016524C is apart of the D_80165230 array

Reply:
I couldn't get either direction to match. This is a weird contradictory variable duplication and array overflow. D_80165230 should technically be size of 8 not 7.

Reply:
yeah I also see that but when I remove D_8016524C and set a size of 8 It's not match so maybe need to be fix later